src/test/java/org/graylog/integrations/s3/S3ScannerFactoryTest.java

```diff
@@ -0,0 +1,10 @@
+package org.graylog.integrations.s3;
+
+public class S3ScannerFactoryTest {
```

Comment:
Should this test class be removed until tests are added?

Reply:
Agreed.  Will remove.